packages/botonic-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@botonic/react",
-  "version": "0.18.0",
+  "version": "0.18.1",
```

Comment:
Since this has been published as alpha, should the new versions contain 'alpha' too?

Reply:
if the PR is approved I will publish the 0.18.1 version